CPPLINT.cfg

```diff
@@ -1,17 +1,19 @@
-# Copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
-#
-# See LICENSE for license information.
-
-# Stop searching for additional config files.
-set noparent
-
-# Limit line length.
-linelength=100
-
-# Ignore the following errors.
-filter=-build/include_subdir
-filter=-build/namespaces
-filter=-readability/todo
-filter=-build/header_guard
-filter=-build/include
-filter=-build/c++11
+# Copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+
+# Stop searching for additional config files.
+set noparent
+
+# Limit line length.
+linelength=100
+
+# Ignore the following errors.
+filter=-build/include_subdir
+filter=-build/namespaces
+filter=-readability/todo
+filter=-build/header_guard
+filter=-build/include
+filter=-build/c++11
+filter=-runtime/references
```

Comment:
I'm fine with this exclusion. I've always found it kind of weird that cpplint doesn't like non-const references since my intuition is that references are safer than pointers.

Reply:
Google c++ style guide also updated to include this, however cpplint wasn't updated.